.buildkite/x-pack/pipeline.xpack.metricbeat.yml

```diff
@@ -360,7 +360,6 @@ steps:
     steps:
       - label: ":ubuntu: x-pack/metricbeat: AWS Tests"
         key: "x-pack-metricbeat-extended-cloud-test"
-        if: build.env("GITHUB_PR_LABELS") =~ /.*aws.*/
```

Comment:
See `L358`, and there is no need to add a new conditional but run always on main or any other branches. This should help with testing things always, rather than on a cron basis:
- https://github.com/elastic/beats/pull/40222

Reply:
👍 Seems to have successfully run: https://buildkite.com/elastic/beats-xpack-metricbeat/builds/15605#0196a073-42d9-4081-ada6-649267d3ce68/210-214